xrpl/models/response.py

```diff
@@ -3,19 +3,24 @@
 
 Represents fields common to all response types.
 """
+
 from __future__ import annotations
 
 import warnings
 from dataclasses import dataclass
 from enum import Enum
 from typing import Any, Dict, Optional, Union
 
+from typing_extensions import Final
+
 from xrpl.models.base_model import BaseModel
 from xrpl.models.required import REQUIRED
 from xrpl.models.transactions import PaymentFlag
 from xrpl.models.transactions.types import TransactionType
 from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
 
+DEFAULT_API_VERSION: Final[int] = 2
```

Comment:
Why is this defined in this file, where it's not used, instead of `request.py`?

Reply:
I was looking for a global constants file, but I couldn't find it.
I've updated the location to requests.py file 👍